dev_scripts/find_images_to_rebuild.py

```diff
@@ -50,5 +58,15 @@
     dry_run=True, **kwargs)
```

Comment:
Just above this, you should be able to get the argument from `args` rather than `sys.argv`.